models.js

```diff
@@ -0,0 +1,23 @@
+const pg = require('pg');
```

Comment:
While this is sufficient for now, I would lean towards using something that facilitates migrations. Check out https://github.com/theoephraim/node-pg-migrate.